azure/terraform/README.md

```diff
@@ -31,10 +31,34 @@ This is used to help generate unique resource names for:
 * Azure storage account name
 Note: these resources must be globally unique across all of Azure.
 
+The template also provides the `deploy_airflow` and `deploy_argo` flags. These are booleans that specify if airflow or argo will be deployed in the kubernetes cluster along with metaflow related services. By default `deploy_argo` is set to __true__ and `deploy_airflow` is set to __false__.
+
 Next, apply the `infra` module (creates Azure cloud resources only).
 
     terraform apply -target="module.infra"
 
+## Orchestrating Metaflow jobs on Kubernetes
+
+The recommended way to orchestrate Metaflow workloads on Kubernetes is via [Argo Workflows](https://docs.metaflow.org/going-to-production-with-metaflow/scheduling-metaflow-flows/scheduling-with-argo-workflows). However, Airflow is also supported as an alternative.
```

Comment:
Shall we make Airflow link to something for folks who don't even know what Airflow is?